### DISCUSSION TITLE
how tu use git lfs

### DISCUSSION THREAD

**Discussion by gameboydathird** [git-lfs/git-lfs#5527] [Q&A]
hi gaizz need help using git lfs trying to upload games to my git pages website

https://gameboydathird.github.io/gitgamesite/

cant use, tried to do the git add filename git commit -m "commit lfs test" then git push origin main also tried git lfs push origin main
 i have done the tutorial alrdy

**Comment by bk2204** [comment]
Hey,

What's the specific problem you're seeing?  In what way is it not working?  Can you include suitable error messages, if any?
